README.md

```diff
@@ -4,6 +4,11 @@ Follow [https://www.mkdocs.org/getting-started/](https://www.mkdocs.org/getting-
 & [https://squidfunk.github.io/mkdocs-material/getting-started/](https://squidfunk.github.io/mkdocs-material/getting-started/)
 
 Run : 
-- `pip install mkdocs mkdocs-material mkdocs-git-revision-date-localized-plugin mkdocs-redirects`
-- `mkdocs serve` in the project root folder
-- Visit  [http://localhost:8000/](http://localhost:8000/)
+ - Via Python
+   - `pip install mkdocs mkdocs-material mkdocs-git-revision-date-localized-plugin mkdocs-redirects`
+   - `mkdocs serve` in the project root folder
+   - Visit  [http://localhost:8000/](http://localhost:8000/)
+ - Via [Docker](https://squidfunk.github.io/mkdocs-material/getting-started/ "more recent methods and docs may very well be available there") (in case you are not a big python fan)
```

Comment:
May be change the docker link to https://hub.docker.com/r/squidfunk/mkdocs-material ?

Reply:
I'm not sure about this one, as a reader I'd much rather have a link to the mkDoc's main doc which explains all that I'm about to read along with official alternatives and a link to the dockerHub page that directly a link to the docker hub page :)

I'm not gonna change that in this PR as to leave this request how I'd like to have the doc on my consumer's perspective but you may, it's your documentation this is simply a suggestion ^^

Your kingdom your rules ;p